### PR TITLE
[TASK][28433] Search

### PR DESCRIPTION
**Pull request by vinhcnt**
https://edu-redmine.sun-asterisk.vn/issues/28433
Screen Shot 2020-08-31 at 08 49 51
Screen Shot 2020-08-31 at 08 49 58

### CODE REVIEW

**Review by thuyptsun — APPROVED** [review]

**Comment by vinhcnt** [comment]
ready

File: app/views/educations/_education.html.erb [hunk 1]
```diff
@@ -5,9 +5,11 @@
     <p class="pb-3 mb-0"><%= education.major %></p>
     <p class="pt-3 border-top mb-0"><%= education.additional_information %></p>
     <p class="pt-3 mb-0"><%= education.certification %></p>
-    <% unless current_page? controller: :profiles, action: :show %>
-      <%= form_for education, method: :delete, remote: true do |f| %>
-        <%= f.submit t(".delete"), class: "btn btn-sm text-danger" %>
+    <% unless current_page?(controller: :profiles, action: :show)  %>
```
Comment (hieplt-0914) [inline app/views/educations/_education.html.erb]: ()

File: app/views/educations/_education.html.erb [hunk 2]
```diff
@@ -5,9 +5,11 @@
...
+      <% if (education.profile.user == current_user) %>
```
Comment (hieplt-0914) [inline app/views/educations/_education.html.erb]: ()

File: app/views/experiences/_experience.html.erb [hunk 3]
```diff
@@ -2,16 +2,18 @@
   <div class="border rounded job-list-box p-4">
     <div class="job-list-desc candidates-profile-exp-desc">
       <div>
-      <% unless current_page?(controller: :profiles, action: :show) %>
-        <%= form_for experience, method: :delete, remote: true do |f| %>
-          <%= f.submit t(".delete"), class:"btn mr-2 btn-sm text-danger float-right" %>
+        <% unless current_page?(controller: :profiles, action: :show) %>
```
Comment (hieplt-0914) [inline app/views/experiences/_experience.html.erb]: ()

File: app/views/experiences/_experience.html.erb [hunk 4]
```diff
@@ -2,16 +2,18 @@
...
+          <% if (experience.profile.user == current_user) %>
```
Comment (hieplt-0914) [inline app/views/experiences/_experience.html.erb]: ()

**Pull request merged**